Rocket.Chat/Controllers/Auth/AuthTableViewController.swift

```diff
@@ -124,10 +124,13 @@ class AuthTableViewController: BaseTableViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+
         title = serverURL.host
 
         guard let settings = serverPublicSettings else { return }
 
+        emailAuthRow.loginButton.isHidden = !settings.isUsernameEmailAuthenticationEnabled
+
         if !settings.isUsernameEmailAuthenticationEnabled {
             emailAuthRow.registerButton.isHidden = true
```

Comment:
@rafaelks we could put the new line inside the `if` block. That way we would evaluate `settings.isUsernameAuthenticationEnabled` just once.